src/log.h

```diff
@@ -0,0 +1,28 @@
+#ifndef BENCHMARK_LOG_H_
+#define BENCHMARK_LOG_H_
+
+#include <ostream>
+
+namespace benchmark {
+namespace internal {
+
+int GetLogLevel();
```

Comment:
Actually are you sure you want an enum here? Glog doesn't use an enum for it's verbose log either.